Kickstarter-iOS/Views/Cells/PledgeAmountCell.swift

```diff
@@ -0,0 +1,128 @@
+import Library
+import Prelude
+import UIKit
+
+final class PledgeAmountCell: UITableViewCell, ValueCell {
+  // MARK: - Properties
+
+  private lazy var amountInputView: AmountInputView = { AmountInputView(frame: .zero) }()
+  private lazy var inputStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var label: UILabel = { UILabel(frame: .zero) }()
+  private lazy var rootStackView: UIStackView = { UIStackView(frame: .zero) }()
+  private lazy var spacer: UIView = {
+    UIView(frame: .zero)
+      |> \.translatesAutoresizingMaskIntoConstraints .~ false
+  } ()
+  private lazy var stepper: UIStepper = { UIStepper(frame: .zero) }()
+
+  // MARK: - Lifecycle
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    _ = self
+      |> \.accessibilityElements .~ [self.label, self.stepper, self.amountInputView]
+      |> \.backgroundColor .~ UIColor.ksr_grey_300
+
+    self.contentView.addSubviewConstrainedToEdges(self.rootStackView)
+
+    self.rootStackView.addArrangedSubview(self.label)
+    self.rootStackView.addArrangedSubview(self.inputStackView)
+    self.inputStackView.addArrangedSubview(self.stepper)
+    self.inputStackView.addArrangedSubview(spacer)
+    self.inputStackView.addArrangedSubview(self.amountInputView)
+
+    self.spacer.widthAnchor.constraint(greaterThanOrEqualToConstant: Styles.grid(3)).isActive = true
+  }
+
+  required init?(coder aDecoder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+    super.traitCollectionDidChange(previousTraitCollection)
+
+    let traitCollection = self.traitCollection
+
+    if previousTraitCollection?.preferredContentSizeCategory
+      != traitCollection.preferredContentSizeCategory {
+      self.configureView(for: traitCollection)
+    }
+  }
+
+  // MARK: - Styles
+
+  override func bindStyles() {
+    super.bindStyles()
+
+    _ = self.label
+      |> labelStyle
+
+    _ = self.rootStackView
+      |> rootStackViewStyle
+
+    _ = self.stepper
+      |> stepperStyle
+  }
+
+  // MARK: - Configuration
+
+  private func configureView(for traitCollection: UITraitCollection) {
+    _ = self.inputStackView
+      |> inputStackViewStyle(self.traitCollection.ksr_isAccessibilityCategory())
+  }
+
+  func configureWith(value: (amount: Double, currency: String)) {
+    self.amountInputView.configureWith(
+      amount: String(format: "%i", value.amount),
+      placeholder: "\(0)",
+      currency: value.currency
+    )
+  }
+}
+
+// MARK: - Styles
+
+func inputStackViewStyle(_ isAccessibilityCategory: Bool) -> ((UIStackView) -> UIStackView) {
```

Comment:
Perhaps we should test these to see that the `UIStackView` that's passed in is configured correctly.